src/main/java/de/doubleslash/keeptime/view/ReportController.java

```diff
@@ -194,6 +198,18 @@ protected void updateItem(final TableRow item, final boolean empty) {
       rootItem.setExpanded(true);
    }
 
+   private void collapseExpandTable(){
+      for (int i=0; i<rootItem.getChildren().size(); i++){
+
+         if( rootItem.getChildren().get(i).isExpanded()){
+            rootItem.getChildren().get(i).setExpanded(false);
+            expandColapseButton.setText("Expand");
+         }else {
+            rootItem.getChildren().get(i).setExpanded(true);
+            expandColapseButton.setText("Collapse");
```

Comment:
setting the text of the same button multiple times in a for loop will (in the end) set the text to the status of the last element in the loop. does not make so much sense.

Reply:
is not in the loop anymore